applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/adjoint_max_stress_response_function.h

```diff
@@ -169,7 +169,7 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) AdjointMaxStressResponseFunct
 
     ModelPart& mrAdjointModelPart;
     std::string mCriticalPartName;
-    Element::Pointer mpTracedElementInAdjointPart;
+    Element::Pointer mpTracedElementInAdjointPart {nullptr};
```

Comment:
didn't know this works

Reply:
Well now that I review it myself, I prefer the more explicit notation with "= nullptr". I changed it. Could you just quickly reapprove? Thanks!

Reply:
I can approve but @armingeiser and @msandre requested changes ...

Reply:
Yeah true. Anyways they also requested changes, so it will not be possible to merge before their approval. So, of course, we wait for them. It is just because you already gave an approval before.